integration-tests/pom.xml

```diff
@@ -332,6 +332,7 @@
                 <module>reactive-messaging-rabbitmq-dyn</module>
                 <module>reactive-messaging-hibernate-reactive</module>
                 <module>reactive-messaging-hibernate-orm</module>
+                <module>rest</module>
```

Comment:
I don't think should add a new module for this

Reply:
Can you point me to where should I put it then?

Reply:
Unless I am missing something, it can be a QuarkusUnitTest in the RESTEasy Reactive server's deployment module